test/federated/test_train_config.py

```diff
@@ -450,10 +441,10 @@ def forward(self, x):
 
     # Create and send train config
     train_config = sy.TrainConfig(model=model, loss_fn=loss_fn, batch_size=2, epochs=1)
-    train_config.send(local_worker)
+    train_config.send(remote_worker)
```

Comment:
Could we use remote_proxy instead of remote_worker? Because the object previously called local_worker is actually local and not remote. Yet it represents a proxy of the remote worker.

Reply:
Please take a look at the changes in #2386 and #2388. They address another aspect of the websocket connection setup. I wasn't aware (forgot) about your changes to these files.